### PR TITLE
log error during st creation

### PR DESCRIPTION
**Pull request by QuentinAmbard**
log exception during statement errors and continue

### CODE REVIEW

File: src/main/scala/com/datastax/gatling/plugin/request/CqlRequestAction.scala [hunk 1]
```diff
@@ -69,43 +69,54 @@ class CqlRequestAction(val name: String,
...
+      stmt.onFailure(err => {
+        handleFailure(session, responseTimeBuilder, err)
+      })
 
-      statsEngine.logResponse(session, name, responseTime.toGatlingResponseTimings, KO, None,
-        Some(s"$tagString - Preparing: ${err.take(50)}"), List(responseTime.latencyIn(MICROSECONDS), "PRE", logUuid))
+      stmt.onSuccess({ stmt =>
+        // global options
+        dseAttributes.cl.map(stmt.setConsistencyLevel)
+        dseAttributes.dynamicCl.map(f => stmt.setConsistencyLevel(f()))
```
Comment (pingtimeout) [inline src/main/scala/com/datastax/gatling/plugin/request/CqlRequestAction.scala]: What's this?

Comment (QuentinAmbard) [reply src/main/scala/com/datastax/gatling/plugin/request/CqlRequestAction.scala]: that's me not merging my branches properly...  It's from the other PR

**Comment by QuentinAmbard** [comment]
@pingtimeout should be fixed now

**Comment by pingtimeout** [comment]
Could you add a unit test for this case?

**Comment by pingtimeout** [comment]
I am wondering why the exception is not caught by the `Validation` object as a `Failure`...

**Comment by bradfordcp** [comment]
@QuentinAmbard @pingtimeout added unit test to this PR. Please review and merge.

**Pull request merged**